modules/nw-multus-ipvlan-object.adoc

```diff
@@ -26,23 +26,23 @@ plugin:
 
 |`type`
 |`string`
-|The name of the CNI plugin to configure: `ipvlan`.
+|Optional: The name of the CNI plugin to configure: `ipvlan`.
 
 |`mode`
 |`string`
-|The operating mode for the virtual network. The value must be `l2`, `l3`, or `l3s`. The default value is `l2`.
+|Optional: The operating mode for the virtual network. The value must be `l2`, `l3`, or `l3s`. The default value is `l2`.
 
 |`master`
 |`string`
-|The Ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
+|Optional: The Ethernet interface to associate with the network attachment. If a `master` is not specified, the interface for the default network route is used.
 
 |`mtu`
 |`integer`
-|Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
 |`ipam`
-|`object`
-|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition.
+|`object` (required unless chained)
+|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition. This required unless chained.
 
 Do not specify `dhcp`. Configuring IPVLAN with DHCP is not supported because IPVLAN interfaces share the MAC address with the host interface.
```

Comment:
I'm missing the notes to understand what chained means. It's in the official doc of cni

Reply:
I added a note on this

Reply:
I meant this notes for ipvlan https://www.cni.dev/plugins/current/main/ipvlan/#notes. The last bullet is about the ipam.

Reply:
did you address this? I still see the previous note about chained plugins.